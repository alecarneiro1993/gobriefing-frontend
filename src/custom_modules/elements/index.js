export { default as Button } from "./Button";
export { default as GradientOverlay } from "./GradientOverlay";
export { default as PageHeading } from "./PageHeading";
export { default as LoadingSpinner } from "./LoadingSpinner";
export { default as Icon } from "./Icon";
export { default as RobotoTypography } from "./RobotoTypography";
export { default as SairaTypography } from "./SairaTypography";
export { default as MobileLogoHeader } from "./MobileLogoHeader";
