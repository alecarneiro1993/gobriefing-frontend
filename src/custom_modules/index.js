export * from './elements';
export * from './form';
export * from './layout';
export * from './toasters';
