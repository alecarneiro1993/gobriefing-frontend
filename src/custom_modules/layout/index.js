export { default as Page } from "./Page";
export { default as Container } from "./Container";
export { default as Row } from "./Row";
export { default as Column } from "./Column";
export { default as OffsetColumn } from "./OffsetColumn";
export { default as Block } from "./Block";
