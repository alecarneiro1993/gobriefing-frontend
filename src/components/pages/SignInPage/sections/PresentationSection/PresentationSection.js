import React from "react";
import { useTranslation } from "react-i18next";

import {
  GradientOverlay,
  SairaTypography,
  RobotoTypography
} from "custom_modules/elements";
import { Block } from "custom_modules/layout";
import {
  PRODUCT_NAME,
  CENTER,
  WHITE,
  LIGHT_WHITE
} from "utils/constants/values";

export default function PresentationSection() {
  const { t } = useTranslation();
  return (
    <GradientOverlay color={{ from: "#667eea", to: "#764ba2" }}>
      <Block>
        <SairaTypography
          gutterBottom
          component="h1"
          variant="h3"
          align={CENTER}
          color={WHITE}
        >
          {PRODUCT_NAME}
        </SairaTypography>
      </Block>
      <Block>
        <RobotoTypography
          gutterBottom
          component="h1"
          variant="h3"
          align={CENTER}
          color={LIGHT_WHITE}
        >
          {t("catch_phrase")}
        </RobotoTypography>
      </Block>
    </GradientOverlay>
  );
}
