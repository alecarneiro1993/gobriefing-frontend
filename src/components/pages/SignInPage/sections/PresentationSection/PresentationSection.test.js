import React from "react";
import { shallow } from "enzyme";

import { GradientOverlay } from "custom_modules";
import PresentationSection from "./PresentationSection";

describe("PresentationSection", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<PresentationSection />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(GradientOverlay)).toHaveLength(1);
  });
});
