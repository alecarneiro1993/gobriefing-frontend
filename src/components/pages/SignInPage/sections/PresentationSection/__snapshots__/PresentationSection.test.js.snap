// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PresentationSection renders without crashing 1`] = `
<Styled(Styled(MuiBox))
  color={
    Object {
      "from": "#667eea",
      "to": "#764ba2",
    }
  }
>
  <Component>
    <Styled(WithStyles(ForwardRef(Typography)))
      align="center"
      color="white"
      component="h1"
      gutterBottom={true}
      variant="h3"
    >
      goBriefing
    </Styled(WithStyles(ForwardRef(Typography)))>
  </Component>
  <Component>
    <Styled(WithStyles(ForwardRef(Typography)))
      align="center"
      color="#E8E8E8"
      component="h1"
      gutterBottom={true}
      variant="h3"
    >
      catch_phrase
    </Styled(WithStyles(ForwardRef(Typography)))>
  </Component>
</Styled(Styled(MuiBox))>
`;
